### DISCUSSION TITLE
Welcome to GitHub Discussions!

### DISCUSSION THREAD

**Discussion by NotMyFault** [IntellectualSites/FastAsyncWorldEdit#938] [General]
Hello folks,

Welcome to the FastAsyncWorldEdit Discussions forum! This is the forum for all things FastAsyncWorldEdit.

Check out some of our categories:

- 🐞If you found a bug, it is best to [submit an issue](https://github.com/IntellectualSites/FastAsyncWorldEdit/issues/new/choose) to ensure our team resolves your issue.
- 💡 Ideas and Feature Suggestions: Share and discuss ideas for new features!
- ❓ If you have a question, [search the wiki](https://github.com/IntellectualSites/FastAsyncWorldEdit-Documentation/wiki) or hop on [our discord server](https://discord.gg/intellectualsites).
- 🆕 Otherwise, start a new discussion.

As a reminder, be respectful and friendly, we want to make this a place that everyone feels welcome! 

If you've got any feedback on the forum, feel free to reply to this thread!

**Comment by ekanemudeme** [comment]
how can i build dashboard for my website

**Reply by NotMyFault** [reply]
Uh, I think you are at the wrong discussions.
